docs/resources/Channel.md

```diff
@@ -225,12 +269,12 @@ Represents a message sent in a channel within Discord.
 
 \*\*\*\* Not all channel mentions in a message will appear in `mention_channels`. Only textual channels that are visible to everyone in a lurkable guild will ever be included. Only crossposted messages (via Channel Following) currently include `mention_channels` at all. If no mentions in the message meet these requirements, this field will not be sent.
 
-\*\*\*\*\* This field is only returned for messages with a `type` of `19` (REPLY). If the message is a reply but the `referenced_message` field is not present, the backend did not attempt to fetch the message that was being replied to, so its state is unknown. If the field exists but is null, the referenced message was deleted.
+\*\*\*\*\* This field is only returned for messages with a `type` of `19` (REPLY) or `21` (THREAD_STARTER_MESSAGE). If the message is a reply but the `referenced_message` field is not present, the backend did not attempt to fetch the message that was being replied to, so its state is unknown. If the field exists but is null, the referenced message was deleted.
 
 ###### Message Types
 
 > warn
-> Type `19` and `20` are only in API v8. In v6, they are still type `0`.
+> Type `19` and `20` are only in API v8. In v6, they are still type `0`.  Type 21 is only in API v9.
```

Comment:
Type 18 is sent in the guild_text channel that a thread was created in, therefore it will be sent in API v8 and below too
Type 21 is sent in the _thread_ and since API v8 & below don't get thread dispatches, they don't get this either